src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -485,6 +485,11 @@
         </summary>
       </Comments>
     </Property>
+    <Property Name="DisposeMethod" Type="IMethodSymbol?">
```

Comment:
It doesn't feel like making this only be populated if it's not `IDisposable.Dispose` is particularly useful, it feels more like an implementation detail bleeding through. I think we should just always populate this, and if it's `IDiposable` then we populate it with `IDisposable.Dispose`.